### PR TITLE
Added presentation link and modified the documentation

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:
-Add presentation link and modified the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 ![screenshot](./screenshots/screen2.png)
 Covid Tracker is site used to check covid updates.
```
Comment (micheaol) [inline README.md:17]: Please ensure that the app has unit tests for several components of the app and some pure functions in Redux code according to the project requirement.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
 ## [Live Demo](https://ncov-site.netlify.app/)
-## [Presentation](#)
+## [Presentation](https://youtu.be/YaC8EksPYMc)
```
Comment (micheaol) [inline README.md:19]: Please ensure that the app has integration tests using React Testing Library according to the project requirement.

* You can perform a mocking data fetching and Redux Store connection.

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@
 ![screenshot](./screenshots/screen2.png)
 Covid Tracker is site used to check covid updates.
 ## [Live Demo](https://ncov-site.netlify.app/)
-## [Presentation](#)
```
Comment (micheaol) [inline README.md:18]: Please ensure that you stick to the design  [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size, and weight.
`Layout: composition and space between elements.`

Also, this is meant to be a mobile app.

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ksigei , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 4]
```diff
@@ -55,6 +55,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-react-hooks": "^4.5.0",
+    "react-test-renderer": "^18.1.0",
     "sass": "^1.52.1",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
```
Comment (brytebee) [inline package.json:61]: ### You have made no changes other than to test files.

image

- [ ] **Required:** You are required to follow the [design guideline](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

- The app has a nav which you have.
- The app has a section containing the world map and the search bar.
- Then the card of cards containing countries and their maps.

Kindly implement them and request another review.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ksigei👋, 

Great work on making the changes 👏 👏

- You've added some tests as requested by the previous reviewer.

You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: package.json (hunk 4 above)
Comment (ksigei) [reply package.json:61]: @brytebee the API that I am using has no maps, neither does it have continents. I am created my web according the API that I am using.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
### Hi  @ksigei :wave:  

Great work, your project is coming along just fine. 👏 

- [ ] Kindly work on your UI design, the borders in the design should not be there.
- [ ] The fonts, make them look better.
- [ ] ... kindly go through the design file again, and try as much as you can, to adapt your design to what is given.

[**Optional**] You can use styled-components to make your styling dynamic.

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @ksigei, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Please update your live version of the project. I'm still seeing your old design on [netlify](https://ncov-site.netlify.app/)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Comment by ksigei** [comment]
@keneogu, the live link comes from the dev branch. When I merge, it will work

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
the-simpsons-homer-simpson

## Suggested changes

- [ ] As requested by the previous reviewer, please try to stick to the proposed design font and colors.

| Screen Shot 2022-06-04 at 19 48 38 |  171836485-7bdbe91c-aaf0-4f3b-8932-f0b8513c21ee |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ksigei** [comment]
@aimemalaika which change, I was asked to share the live link

**Comment by aimemalaika** [comment]
@ksigei Kindly follow the design guide. the first card of the grid is full width, some text are not fitting in their card, use the correct font, the search bar is not centered and the proposed design has no margin or padding around the cards

Work on those request the resubmit for review. 👍🏽

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @ksigei , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Kudos to you!! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**